src/components/Tooltip/Tooltip.tsx

```diff
@@ -56,113 +59,268 @@ const paddingsBottom = {
     large: "tw-pb-4",
 };
 
-const TooltipComponent: ForwardRefRenderFunction<HTMLDivElement, TooltipProps> = (
-    {
-        content,
-        tooltipIcon,
-        heading,
-        headingIcon,
-        linkUrl,
-        linkLabel,
-        brightHeader,
-        buttons,
-        tooltipAriaProps = {},
-        style,
-        children,
-        popperAttributes,
-    },
-    tooltipContainerRef,
-) => {
+export enum TooltipPosition {
+    Top = "top",
+    Right = "right",
+    Bottom = "bottom",
+    Left = "left",
+}
+
+export enum TooltipAlignment {
+    Start = "Start",
+    Middle = "Middle",
+    End = "End",
+}
+
+const placementMap: Record<string, VariationPlacement> = {
+    ["top-Start"]: "top-start",
+    ["top-End"]: "top-end",
+    ["bottom-Start"]: "bottom-start",
+    ["bottom-End"]: "bottom-end",
+    ["left-Start"]: "left-start",
+    ["left-End"]: "left-end",
+    ["right-Start"]: "right-start",
+    ["right-End"]: "right-end",
+};
+
+export const Tooltip: FC<TooltipProps> = ({
+    content,
+    tooltipIcon,
+    heading,
+    headingIcon,
+    linkUrl,
+    linkLabel,
+    brightHeader,
+    buttons,
+    children,
+    position = TooltipPosition.Bottom,
+    alignment = TooltipAlignment.Middle,
+    withArrow,
+    flip = true,
+    triggerRefElement,
+    hoverDelay = 200,
+}) => {
     const linkRef = useRef<HTMLAnchorElement | null>(null);
     const { linkProps } = useLink({}, linkRef);
-
     const { isFocusVisible, focusProps } = useFocusRing();
+    const hasLargePaddingTop = linkUrl || brightHeader || buttons || heading || headingIcon;
```

Comment:
`useMemo` so it doesn't get recomputed for each render in case any other props changes